src/backend/__mocks__/electron.ts

```diff
@@ -16,17 +16,20 @@ const dialog = {
   showMessageBox: jest.fn()
 }
 
-const app = {
+class App extends EventEmitter {
   // app override
-  getPath: jest.fn().mockImplementation((path: string) => {
+  getPath = jest.fn().mockImplementation((path: string) => {
     return join(appBasePath, path)
-  }),
+  })
   getVersion(): string {
     return '1.0.0'
-  },
-  requestSingleInstanceLock: jest.fn().mockReturnValue(true)
+  }
+
+  requestSingleInstanceLock = jest.fn().mockReturnValue(true)
 }
 
+const app = new App()
+
```

Comment:
why did you need to do this?

Reply:
`app.on('web-contents-created', openNewBrowserGameWindowListener)` needs to be mocked or the tests fail

app inherits from `EventEmitter` in electron